### DISCUSSION TITLE
[FR] Choix du client

### DISCUSSION THREAD

**Discussion by LyonelB** [baimard/gestion#352] [Ideas]
Actuellement, quand on cherche un client lors de la création d'un devis, la liste affiche les prénoms et noms des clients. Est-il possible d'afficher le nom de la nom au lieu du prénom/nom ?

**Comment by baimard** [comment]
Le nom de l'entreprise tu veux dire ?

**Comment by LyonelB** [comment]
Oui

**Comment by baimard** [comment]
Bonjour,
J'ai réfléchit à cette fonctionnalité, je suis entièrement d'accord avec toi. Je vais trouver une alternaive, par exemple nom de l'entreprise affiché c'est plus logique et quand on passe la souris dessus, nom et prenom de contact s'affiche.

Cela te conviendrait il ?

**Comment by LyonelB** [comment]
> Bonjour, J'ai réfléchit à cette fonctionnalité, je suis entièrement d'accord avec toi. Je vais trouver une alternaive, par exemple nom de l'entreprise affiché c'est plus logique et quand on passe la souris dessus, nom et prenom de contact s'affiche.
> 
> Cela te conviendrait il ?

Oui !
Je te partage ici les autres idées que j'avais : 
- pour choisir une numérotation automatique pour les devis/factures incluant la date.
- dans "configuration", ajouter un bloc text pour pouvoir ajouter plusieurs lignes.  Exemple de texte que je souhaite ajouter :

```
CLAUSE DE RÉSERVE DE PROPRIÉTÉ : Conformément à la loi 80.335 du 12 mai 1980, nous réservons la propriété des produits et marchandises, objets des présents débits, jusqu'au paiement de l'intégralité du prix et de ses accessoires. En cas de non paiement total ou partiel du prix de l'échéance pour quelque cause que ce soit, de convention expresse, nous nous réservons la faculté, sans formalités, de reprendre matériellement possession de ces produits ou marchandises à vos frais, risques et périls.
Pénalité de retard : 3 fois le taux d'intérêt légal après date échéance. Escompte pour règlement anticipé : 0% (sauf condition particulière définie dans les conditions de règlement)
Le montant de l'indemnité forfaitaire pour frais de recouvrement prévue en douzième alinéa de l'article L441-6 est fixé à 40 Euros en matière commerciale.
```

**Comment by baimard** [comment]
Il y a déjà une zone pour ça. c'est pour les mentions légales, pour les idées plutôt utiliser discussion et laisser le ticket pour que je suive ce que je fais.

**Comment by ourssansplus** [comment]
> Bonjour, J'ai réfléchit à cette fonctionnalité, je suis entièrement d'accord avec toi. Je vais trouver une alternaive, par exemple nom de l'entreprise affiché c'est plus logique et quand on passe la souris dessus, nom et prenom de contact s'affiche.
> 
> Cela te conviendrait il ?

Bonjour ! Cette option est valable pour un projet en BtoC ! Pour une entreprise qui s'adresse aux particuliers, l'affichage Nom/prénom est plus adapté ! Faudrait pouvoir choisir !
